nservicebus/serialization/index.md

```diff
@@ -35,11 +35,13 @@ The [Newtonsoft JSON Serializer](newtonsoft.md) provides an effective general-pu
 
 The default serializer used in NServiceBus projects is the custom [XmlSerializer](xml.md).
 
+NOTE: If you are transitioning to a new serializer then messages that are currently 'in flight' are formatted using the previous format. These messages cannot be deserialized prior to NServiceBus 6. Make sure *all* queues of all endpoints are drained/empty. **This includes [Saga Timeout](/nservicebus/sagas/timeouts.md) and [Deferred/Delayed](/nservicebus/messaging/delayed-delivery.md) messages via timeout persistence.** Live migration is only support from NServiceBus 6 and higher.
```

Comment:
I'd prefer to see this note be in a partial so that we can have different content for NSB 6+. Right now it's confusing that the "These messages cannot be deserialized prior to NServiceBus 6" sentence is right in the middle of the warning, but it doesn't actually explain why this is true, and assumes you'll understand that it's referring to the additional serilzliers section below... which is in a 6+ partial.

Creating two versions of this next will clarify things. In the < 6 version, you can be clearer that this isn't supported, so you need to ensure there are no in-flight messages.

For the 6+ version, you can just say something about needing to add an additional serializer to handle this.